src/assets/Yaml/slackInfo.yml

```diff
@@ -0,0 +1,13 @@
+---
+team: TC61JSPUZ
+channel:
+  general: CPRDXHHD4
+  announcements: CC5S86GHE
+  fundraisingCommittee: CCUUWRU1Y
+  motorshpe: CP5HC76TD
+  projects: CNATZRD9C
+  shpejr: GCWFE5630
+  mentorshpe: C016KGD2WHH
+  computereng: GN4U8FPDK
+  projectleads: GNBH9T4UC
```

Comment:
I put them there in case the committees wanted to link to slack. Should I take them off?

Reply:
I think it's fine, but (1) people won't find a private channel if they're not in it and (2) we don't have all of the private channel codes, so there are a couple missing. Moreover `computereng` isn't a committee and I'm afraid SHPE will keep changing channels, so it would be safer to only have the default ones.